apps/doc/en-us/3.2-CALLBACK_SCHEMA.md

```diff
@@ -542,5 +544,48 @@ Todo
     // Only ArkPlanner and Lolicon formats are supported. More websites will be supported in future.
     ```
 
+- `OperBoxInfo`<br>
+    Recognition result of operator box
+
+    ```jsonc
+    // Example of corresponding details field
+    "done": bool,       // Has the recognition been completed? False indicates that the recognition is still in progress (data during the process)
```

Comment:
// Whether the recognition has been completed, false means it is still in progress (data during the process)